src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/Messages.properties

```diff
@@ -241,7 +241,7 @@ pscanalpha.imagelocationscanner.alerttitle = Image Contains Embedded Location In
 pscanalpha.imagelocationscanner.alertDetailPrefix = This image contains the following embedded location information:
 pscanalpha.imagelocationscanner.desc = The image was found to contain embedded location information, such as GPS coordinates.  Depending on the context of the image in the website, this information may expose private details of the users of a site.  For example, a site that allows users to upload profile pictures taken in the home may expose the home's address.  
 pscanalpha.imagelocationscanner.refs = http://www.veggiespam.com/ils/
-pscanalpha.imagelecationscanner.soln = Before allowing images to be stored on the server and/or transmitted to the browser, strip out the embedded location information from image.  This could mean removing all Exif data or just the GPS component.
+pscanalpha.imagelocationscanner.soln = Before allowing images to be stored on the server and/or transmitted to the browser, strip out the embedded location information from image.  This could mean removing all Exif data or just the GPS component.
```

Comment:
Worth updating this (and other messages) to match the newer version.

Reply:
You mean just update the content to make it more relevant for recent examples?

Reply:
No, just copy the new contents from the `ILS` class (e.g. `ILS.pluginName`, `ILS.alertTitle`...).

Reply:
Oh, I didn't even notice them. Will do...